### PR TITLE
Oxidation

### PR DESCRIPTION
**Pull request by mack-stefan**

### CODE REVIEW

File: ababot/bot/src/commands/dice.rs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                max = option
+                    .value
+                    .as_ref()
+                    .map(|v| v.as_i64())
+                    .flatten()
+                    .unwrap_or(100);
+            }
+        }
+        let mut rng = thread_rng();
+        rng.gen_range(min..max).to_string()
```
Comment (henriklovhaug) [inline ababot/bot/src/commands/dice.rs:43]: Panics at illegal range

File: ababot/bot/src/commands/mod.rs [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// dir_macros::import! {"bot/src/commands"}
```
Comment (henriklovhaug) [inline ababot/bot/src/commands/mod.rs]: Should we leave unused code as refrence or remove it?

File: ababot/dir_macros/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        directory,
+        rust_path,
+        function_name,
+    } = parse_macro_input!(input as InvocationTarget);
+
+    let dir = match fs::read_dir(directory.value()) {
+        Ok(dir) => dir,
+        Err(e) => panic!("{}", e),
+    };
+
```
Comment (henriklovhaug) [inline ababot/dir_macros/src/lib.rs:83]: Comment explaining output

**Review by henriklovhaug — CHANGES_REQUESTED** [review]

File: ababot/dir_macros/src/lib.rs (hunk 3 above)
Comment (henriklovhaug) [reply ababot/dir_macros/src/lib.rs:83]: Comment ment to highlight following code, not the preceding

File: ababot/bot/src/commands/dice.rs (hunk 1 above)
Comment (henriklovhaug) [reply ababot/bot/src/commands/dice.rs:43]: Patched

File: ababot/bot/src/commands/mod.rs (hunk 2 above)
Comment (ghost) [reply ababot/bot/src/commands/mod.rs]: We can probably remove it until we find a way to work with it. It's worthless if we don't get intelisens

File: ababot/dir_macros/src/lib.rs (hunk 3 above)
Comment (ghost) [reply ababot/dir_macros/src/lib.rs:83]: What

Comment (henriklovhaug) [reply ababot/dir_macros/src/lib.rs:83]: Moved to next iter

**Review by henriklovhaug — APPROVED** [review]
Looks good for initial deployment

**Pull request merged**